backend/photos/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import mock
+import os
+from django.conf import settings
+
+
+@pytest.fixture(scope='session')
+def django_db_modify_db_settings(django_db_modify_db_settings,):
+    os.environ['ENV'] = 'test'
+    settings.DATABASES['default'] = {
+        'ENGINE':   'django.db.backends.sqlite3',
+        'NAME':     ':memory:'
+    }
+
+
+@pytest.fixture(autouse=True)
+def mock_redis(request):
```

Comment:
In the mock_redis fixture, it won't actually perform locking will it? Do we need to worry about this if tests are being run in parallel?